pyvista/plotting/plotter.py

```diff
@@ -1575,7 +1575,22 @@ def length(self):  # numpydoc ignore=RT01
 
     @property
     def center(self):  # numpydoc ignore=RT01
-        """Return the center of the active renderer."""
+        """Return the center of the active renderer.
+
+        Returns
+        -------
+        list[float, float, float]
```

Comment:
```suggestion
        list[numpy.float64, numpy.float64, numpy.float64]
```
```python
>>> type(pl.center)
list
>>> type(pl.center[0])
numpy.float64
>>> type(pl.center[1])
numpy.float64
>>> type(pl.center[2])
numpy.float64
```

Reply:
In that case wouldn't `Plotting.bounds` also return `tuple[numpy.float64, numpy.float64, numpy.float64, numpy.float64, numpy.float64, numpy.float64]`

```py
>>> type(pl.bounds[0]) 
<class 'numpy.float64'>
```

Reply:
@Arnav-2004 Good catch. Could you also fix it too?